recipes/deploy-engage.rb

```diff
@@ -0,0 +1,101 @@
+# Cookbook Name:: mh-opsworks-recipes
```

Comment:
Lots of duplication amongst these deploy recipes, once they're relatively stable I think a refactor is in order.